src/commands/env.js

```diff
@@ -67,4 +67,25 @@ async function importEnv (params) {
   Logger.println('Environment variables have been set');
 };
 
-module.exports = { list, set, rm, importEnv };
+async function importEnvVars (params) {
+  const [envNames] = params.args;
+  const { alias } = params.options;
+
+  for (const envName of envNames) {
+    const nameIsValid = validateName(envName);
+    if (!nameIsValid) {
+      throw new Error(`Environment variable name ${envName} is invalid`);
+    }
+  }
```

Comment:
Do we need to validate env var names here?

I mean, we're about to try to read them from current env so if they aren't valid, they won't be in the current env.

I guess it's a good way to fail fast and help a user that maybe made a typo in the command.

I say OK but maybe we would like to throw an error with all the bad names.

Reply:
The idea here was to stay as close to importEnv as possible, and if we're launched from somewhere else than a shell, the env might not be as strict as we think

Reply:
Good catch.